sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java

```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
+     */
     Mono<DatabaseAccountListKeysResult> listKeysAsync();
 
-    /** @return the read-only access keys for the specified Azure CosmosDB database account */
+    /**
+     * Gets the read-only access keys for the specified Azure CosmosDB database account.
+     *
+     * @return the read-only access keys for the specified Azure CosmosDB database account
+     */
     DatabaseAccountListReadOnlyKeysResult listReadOnlyKeys();
 
-    /** @return the read-only access keys for the specified Azure CosmosDB database account */
+    /**
+     * Gets the read-only access keys for the specified Azure CosmosDB database account.
+     *
+     * @return the read-only access keys for the specified Azure CosmosDB database account
+     */
     Mono<DatabaseAccountListReadOnlyKeysResult> listReadOnlyKeysAsync();
 
-    /** @return the connection strings for the specified Azure CosmosDB database account */
+    /**
+     * Gets the connection strings for the specified Azure CosmosDB database account.
+     *
+     * @return the connection strings for the specified Azure CosmosDB database account
+     */
     DatabaseAccountListConnectionStringsResult listConnectionStrings();
 
-    /** @return the connection strings for the specified Azure CosmosDB database account */
+    /**
+     * Gets the connection strings for the specified Azure CosmosDB database account.
+     *
+     * @return the connection strings for the specified Azure CosmosDB database account
+     */
     Mono<DatabaseAccountListConnectionStringsResult> listConnectionStringsAsync();
 
-    /** @return the list of Azure Cosmos DB SQL databases */
+    /**
+     * Gets the list of Azure Cosmos DB SQL databases.
+     *
+     * @return the list of Azure Cosmos DB SQL databases
+     */
     List<SqlDatabase> listSqlDatabases();
 
-    /** @return the list of Azure Cosmos DB SQL databases */
+    /**
+     * Gets the list of Azure Cosmos DB SQL databases.
+     *
+     * @return the list of Azure Cosmos DB SQL databases
+     */
     PagedFlux<SqlDatabase> listSqlDatabasesAsync();
 
-    /** @return whether write is enabled for multiple locations or not */
+    /**
+     * Check whether write is enabled for multiple locations.
+     *
+     * @return whether write is enabled for multiple locations or not
+     */
     boolean multipleWriteLocationsEnabled();
 
-    /** @return whether cassandra connector is enabled or not. */
+    /**
+     * CHeck whether cassandra connector is enabled.
+     *
+     * @return whether cassandra connector is enabled or not.
+     */
     boolean cassandraConnectorEnabled();
 
-    /** @return the current cassandra connector offer. */
+    /**
+     * Gets the current cassandra connector offer.
+     *
+     * @return the current cassandra connector offer.
+     */
     ConnectorOffer cassandraConnectorOffer();
 
-    /** @return whether metadata write access is disabled or not. */
+    /**
+     * Check whether metadata write access is disabled.
```

Comment:
```suggestion
     * Checks whether metadata write access is disabled.
```

Reply:
Fixed by the [commit#a8c40e1](https://github.com/Azure/azure-sdk-for-java/pull/43524/commits/a8c40e15e3cfe24f29ba518652c5b7baa5c057b7)